learn/experimental/vector_search.mdx

```diff
@@ -0,0 +1,14 @@
+
+# Vector search
+
+Vector search is an experimental technology that uses Large Language Models to retrieve search results. This feature is useful when you are interested in results based on the meaning and context of a query, such as suggesting related products in webshop or articles in an online magazine.
```

Comment:
```suggestion
Vector search is an experimental technology that uses machine learning to retrieve search results. This feature is useful when you are interested in results based on the meaning and context of a query, such as suggesting related products in webshop or articles in an online magazine.
```

My understanding is that large language models need vector search to function, but vector search can be useful even without a large language model. Take or leave this suggestion as you like.